src/style.css

```diff
@@ -1 +1,63 @@
-/* Styles go here */
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto-Medium.ttf);
```

Comment:
you can coonect fonts via google fonts, no need to load them all
https://fonts.google.com/specimen/Roboto
![image](https://user-images.githubusercontent.com/37161315/215461130-a128ffbb-0304-4a2d-8357-064eaed46173.png)